spec/factories/notes.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :note do
+    note { "MyText" }
```

Comment:
You can use `Faker::Lorem` for the contents: https://github.com/faker-ruby/faker/blob/master/doc/default/lorem.md